pombola/search/templates/search/location.html

```diff
@@ -6,35 +6,40 @@
 {% endblock %}
 
 {% block search_form %}
-  <input type="text" name="q" value="{{ query }}" id="id_q" />
+  <div class="inline-search-box">
+    <input type="text" name="q" value="{{ query }}" id="id_q" placeholder="Enter your location&hellip;"/>
+    <input type="submit" value="Search" class="button">
+  </div>
 {% endblock %}
 
 {% block search_results %}
-  {% compressed_js 'google-map' %}
-
-  <div class="geocoded_results">
-
-    <div id="map_canvas">&nbsp;</div>
-
-    <ul>
-      {% for result in geocoder_results %}
-        <li>
-          <a href="/place/latlon/{{ result.latitude }},{{result.longitude }}/">
-            {{ result.address }}
-          </a>
-        </li>
-        <script>
-          markers_to_add.push({
-            lat: {{ result.latitude }},
-            lng: {{ result.longitude }},
-            url: "/place/latlon/{{ result.latitude }},{{result.longitude }}/",
-            name: "{{ result.address }}"
-          });
-        </script>
-      {% endfor %}
-    </ul>
-
-    <br style="clear: both;">
-
-  </div>
+  {% if geocoder_results %}
```

Comment:
It would be nice to include an error message saying, "The location 'foobar' couldn't be found" or something like that when there are no results.